lib/action_cable/channel/naming.rb

```diff
@@ -0,0 +1,21 @@
+module ActionCable
+  module Channel
+    module Naming
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Returns the channels's name, underscored, without the <tt>Channel</tt> ending.
```

Comment:
We should clarify that module namespaces are separated with colons.

(Also `channels's name` could be `the name of the channel`.)